autohoot/utils.py

```diff
@@ -146,8 +146,10 @@ def getchar(self):
         if self.char == 'z':
             self.char = 'A'
         elif self.char == 'Z':
-            logging.info('Run out of characters.')
-            raise NotImplementedError
+            # This is consistent with Opt_einsum
+            # https://github.com/dgasmith/opt_einsum/blob/master/opt_einsum/parser.py#L53
+            logging.info('Run out of letters, start using unicode characters.')
+            self.char = chr(192)
```

Comment:
Can you add more comment and/or rename the CharacterGetter?

At least saying it supports a-zA-Z and then Unicode?

And do we have a limit on the unicode? Can we raise an error when it reaches the limit?

Reply:
done